modules/config.py

```diff
@@ -231,6 +237,9 @@ def config(self, event):
             elif result.result == ConfigResults.Removed:
                 event["stdout"].write("Unset setting '%s'%s" %
                     (setting.lstrip("-"), for_str))
+            elif result.result == ConfigResults.Unchanged:
+                event["stdout"].write("Config '%s'%s unchanged" %
```

Comment:
Should I have this as `'%s'` or `'%s'%s`

Reply:
`'%s'%s` is correct!